### PR TITLE
https://polkadot.subscan.io/extrinsic/22658128-2

### PR DESCRIPTION
**Pull request by asggWa**
treebtc:112

https://polkadot.subscan.io/extrinsic/22658128-2

### CODE REVIEW

**Comment by stepanLav** [comment]
Hello!
Thank you for the request, could you provide a proof of ownership for that account? You can use our tutorial for the details
https://docs.novawallet.io/nova-wallet-wiki/opengov/delegate-registry

File: registry/polkadot.json [hunk 1]
```diff
@@ -413,4 +413,12 @@
         "longDescription": "Pirate navigating the seas of OpenGov 🏴‍☠️—voting on cool stuff whenever I can. I run the largest Polkadot content portal & ecosystem wiki too - https://dablock.com",
         "isOrganization": false
     }
+    {
+    "address": "14fp9FUSaeRpLJ8i7pDmNy7zVZNiZnNmDJ1bHEaZpXPqb1AX",
+    "name": "treebtc", 
+    "image": "https://raw.githubusercontent.com/asggWa/opengov-delegate-registry/refs/heads/master/images/20240923_083455_0000.png"    ,
```
Comment (stepanLav) [inline registry/polkadot.json:419]: also, please, replace the path to novasamatech repo
```suggestion
    "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/20240923_083455_0000.png",
```

**Comment by asggWa** [comment]
polkadot.subscan.io/extrinsic/22658128-2

**Review by asggWa — COMMENTED** [review]
treebtc:112
https://polkadot.subscan.io/extrinsic/22658128-2

**Comment by asggWa** [comment]
ok

**Review by asggWa — COMMENTED** [review]
datos añadidos

File: registry/polkadot.json (hunk 1 above)
Comment (asggWa) [reply registry/polkadot.json]: did it

File: registry/polkadot.json [hunk 2]
```diff
@@ -420,5 +420,13 @@
...
     "isOrganization": true
-  }
+  },
+  {
+    "address": "14fp9FUSaeRpLJ8i7pDmNy7zVZNiZnNmDJ1bHEaZpXPqb1AX",
+    "name": "treebtc", 
+    "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/20240923_083455_0000.png"    ,
```
Comment (stepanLav) [inline registry/polkadot.json:427]:
```suggestion
    "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/20240923_083455_0000.png",
```

**Review by stepanLav — APPROVED** [review]

**Pull request merged**